### PR TITLE
HIVE-1 add hive scripts

### PR DESCRIPTION
**Pull request by danilolazurenko**

### CODE REVIEW

File: hurma/hive_export_jobs/create_funds_table.hql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+create database if not exists testdb;
+use testdb;
```
Comment (danilolazurenko) [inline hurma/hive_export_jobs/create_funds_table.hql]: crunchbase

File: hurma/hive_export_jobs/create_funds_table.hql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+create database if not exists testdb;
```
Comment (danilolazurenko) [inline hurma/hive_export_jobs/create_funds_table.hql]: create db in another file

File: hurma/hive_export_jobs/create_funds_table.hql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  updated_at timestamp,
+  entity_uuid string,
+  entity_name string,
+  entity_type organization,
+  announced_on date,
+  raised_amount_usd int,
+  raised_amount int,
+  raised_amount_currency_code string
+)
+row format delimited
```
Comment (danilolazurenko) [inline hurma/hive_export_jobs/create_funds_table.hql]: use parquet

File: scripts/load_data_to_hive.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (danilolazurenko) [inline scripts/load_data_to_hive.sh]: copy data to hdfs

File: scripts/load_data_to_hive.sh [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+# author     : dl
+# usage      : if in shared directory `source load_data_to_hive.sh`
+
+file_input=$1
+table_name=$2
+
+if ! [ -f file_input ] ; then
+	echo "file_input does not exist"
+
+namenode_path = hdfs://namenode:8020/user/hive/warehouse/testdb.db/$table_name
```
Comment (danilolazurenko) [inline scripts/load_data_to_hive.sh]: pass conf

File: hurma/hive_sql/create_schema.hql [hunk 6]
```diff
@@ -0,0 +1,83 @@
+create database if not exists crunchbase;
```
Comment (vlazurenko2-ias) [inline hurma/hive_sql/create_schema.hql]: which schema? naming

File: hurma/hive_sql/insert_and_deduplicate_parent_children_org_data.hql [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+       tmp.uuids,
+       tmp.count
+FROM   (SELECT parent_uuid,
+               names,
+               uuids,
+               count,
+               Row_number()
+                 OVER ( partition by ID
+                   ORDER BY parent_uuid, names, uuids) AS rno
+        FROM   parent_children_orgs) as tmp
```
Comment (vlazurenko2-ias) [inline hurma/hive_sql/insert_and_deduplicate_parent_children_org_data.hql]: bad naming. use t or meaningfull

File: scripts/load_org_data_to_hive_and_transform_it.sh [hunk 8]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (vlazurenko2-ias) [inline scripts/load_org_data_to_hive_and_transform_it.sh]: it's always better to have a separate load and transform the flow 
bad naming again. what if you will have another transformation

File: scripts/set_hive_config.sh [hunk 9]
```diff
@@ -0,0 +1,38 @@
+hive -e "
```
Comment (vlazurenko2-ias) [inline scripts/set_hive_config.sh]: what is the scope for these settings? session of global?

File: hurma/hive_sql/create_schema.hql (hunk 6 above)
Comment (danilolazurenko) [reply hurma/hive_sql/create_schema.hql]: https://stackoverflow.com/questions/22144685/create-a-schema-in-hive
The uses of SCHEMA and DATABASE are interchangeable – they mean the same thing

**Pull request merged**